### PR TITLE
Build updates

### PR DESCRIPTION
**Pull request by benjaoming**
Gathering ingredients from:

 * Current `master` branch
 * A `yarn install` + `update_perseus.sh` execution on repo

What I wonder is: Which of these sides in the diff should be released?

The changes here are **unreleased** in kolibri-exercise-perseus-plugin 0.9.1.

Are these changes because of an old version of Node on my side or because branches have differed on our local builds?

### CODE REVIEW

File: kolibri_exercise_perseus_plugin/assets/src/constants.js [hunk 1]
```diff
@@ -1,2 +1 @@
-const ConfigFileName = 'KAthJax-730d56e87e9c926b91584f6030314815.js';
-module.exports = { ConfigFileName };
+
```
Comment (benjaoming) [inline kolibri_exercise_perseus_plugin/assets/src/constants.js:1]: This seems to be a problem with the script, this file shouldn't be empty.

**Comment by benjaoming** [comment]
Maybe they are just source files that were built and released, and then the source files are missing in the repo?

**Comment by benjaoming** [comment]
Verifying manually by downloading and solving some exercises then :)

**Comment by benjaoming** [comment]
Okay staring and staring at this, I cannot find a single ACTUAL change, it's just reordering and syntactic sugar.

**Comment by radinamatic** [comment]
Well, at least it's sweet! 🙂

**Comment by indirectlylit** [comment]
this says "Gathering ingredients from current master branch", but that doesn't seem accurate? (See #138)

**Review by indirectlylit — CHANGES_REQUESTED** [review]
@benjaoming  I don't quite understand these changes either, but from a cursory examination they do not look correct. It looks kind of like the revert-patch of a `prettier` run, or something.

**Comment by benjaoming** [comment]
@indirectlylit yes I couldn't find anything substantial in the changes

re:

> this says "Gathering ingredients from current master branch", but that doesn't seem accurate?

No, I firstly got `master` pulled in, but then got confused about whether `0.9.1` was built from master or not because the version bump was present in both.

So I'm still confused about which branch was released finally :)

**Comment by indirectlylit** [comment]
I'm fairly sure 0.9.1 was built from `master` because I told @DXCanas and @lyw07:

> make sure to pull the latest perseus `master`
> `master` is the default branch and my last PR accidentally went there and not the release branch

**Pull request closed without merging**